docs/README.md

```diff
@@ -28,31 +28,31 @@ In your theme's `manifest.json`, add the Search Result app as a dependency:
-| `store.not-found#search`    | When configured, it displays a 404 error message whenever the server is not able to return what the browser request was or when it is configured to not handle that request.  |  
-| `search-products-count-per-page` | Displays the total number of products being displayed in the search results page. | 
-| `search-products-progress-bar` | Displays a progress bar of products being displayed in the search results page. |
-| `order-by.v2`            | Allows users to choose the product ordination in the search results page.  | 
-| `filter-navigator.v3`        | Allows users to apply different filters to the search. On mobile, renders a button that, when clicked on, displays all available filters in a sidebar. | 
-| `total-products.v2`        | Displays the total amount of products found for that search. | 
-| `search-title.v2`         | Displays a title for the search that was done. |                                                                                              |
-| `search-fetch-more`         | Displays the "Show More" button. This button isn't rendered when the user is on the last page. |                                                                                              |
-| `search-fetch-previous`         | Displays the "Show Previous" button. This button isn't rendered when the user is on the first page. |                                                                                              |
-| `search-products-count-per-page`         | Displays the quantity of products currently on the page. |                                                                                              |
-| `sidebar-close-button`         | Displays an `X` button on the filter sidebar on mobile. |                                                                                              |
-| `search-title.v2`         | Displays a title for the search that was done. |         
-
-:information_source: The Search Result app data may be displayed on **search pages** (`store.search`) or any other desired page. When added to the search page, the block that is used must be the `search-result-layout`, since it fetches data provided by the template's current search context. If you want to add the app to another page, the block that must be used is the `search-result-layout.customQuery`. 
+| Block name                         | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```

Comment:
Hey, @vitorflg! There is no need to add such space to the table. I believe it makes the doc too hard to read (and edit) and at the end of the day, there is no difference in the rendering. 

This applies to all tables below!